packages/firebase_messaging/README.md

```diff
@@ -143,6 +143,15 @@ To integrate your plugin into the iOS part of your app, follow these steps:
 
 1. Follow the steps in the "[Upload your APNs certificate](https://firebase.google.com/docs/cloud-messaging/ios/client#upload_your_apns_certificate)" section of the Firebase docs.
 
+1. Add the following lines to the `(BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions`
+method In the `AppDelegate.m` of your iOS project.
+
+```objectivec
+if (@available(iOS 10.0, *)) {
+  [UNUserNotificationCenter currentNotificationCenter].delegate = (id<UNUserNotificationCenterDelegate>) self;
+}
+```
+
```

Comment:
Alternatively, if you are using Swift over Objective C, add the following lines to the 
`application(
    _ application: UIApplication,
    didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?
  ) -> Bool` method in the `AppDelegate.swift` of your iOS project.

```swift
    if #available(iOS 10.0, *) {
        UNUserNotificationCenter.current().delegate = self as? UNUserNotificationCenterDelegate
    }
```

Reply:
Thanks for pointing this out! I added a swift example.